webservices/db.py

```diff
@@ -26,3 +26,18 @@ def htsql_conn():
         htsql_conn_string = sqla_conn_string().replace('postgresql', 'pgsql')
         g._htsql_conn = HTSQL(htsql_conn_string)
     return g._htsql_conn
+
+def as_dicts(data):
```

Comment:
If you're moving this here (seems like it makes more sense), remember to remove the version in `webservices/resources.py`